src/fmu/dataio/providers/objectdata/_provider.py

```diff
@@ -204,10 +202,8 @@ def get_spec(self) -> dict[str, Any]:
         logger.info("Get spec for dictionary")
         return {}
 
-    def get_bbox(self) -> dict[str, Any]:
+    def get_bbox(self) -> None:
         """Derive data.bbox for dict."""
```

Comment:
I did not know if was possible to have a function without a body.. thought `pass` was needed 🙂

Reply:
As long as a docstring is there python is happy 😄

Reply:
I noticed the same a while ago, by accident. Had started writing a test function but never got further than the definition and the docstring, but in the end forgot all about it - and the tests ran fine for months until I discovered the error :)